### ISSUE TITLE
Update EquipmentType and EquipmentClassType to align with updated 950002

### ISSUE THREAD

**Issue by racheliurui** [ETPartners/b2mmlv7beta#30]
### MESA

https://github.com/MESAInternational/B2MML-BatchML/issues/37

### Background and Solution
_Equipment.xsd_
_complexType name = "EquipmentType"_
> CHANGE: element name = "EquipmentLevel"      type = "HierarchyScopeType"   minOccurs = "0" /> 
TO: type = type = "EquipmentLevelType" minOccurs = "0" />
> ADD element name ="PhysicalAssetID" type = "IdentifierType" minOccurs = "0"/>
> ADD element name = "OperationalLocationType" type = "ResourceLocationTypeType"

_complexType name = "EquipmentClassType">_
> CHANGE: element name = "EquipmentLevel"      type = "HierarchyScopeType"   minOccurs = "0" /> 
TO: type = "EquipmentLevelType" minOccurs = "0" />

### Supporting Document
_ISA-950002 JWG5 CDV01 version (2019 12)_
Clause 5.5.2 Equipment class
Table 40 – Equipment class attributes
Clause 5.5.4 Equipment
Table 43 – Equipment relationship roles
Table 44 – Equipment attributes

### Impacted Type
_B2MML-Equipment.xsd_
- EquipmentType
- EquipmentClassType

**Comment by Dennis-Brandl** [comment]
I agree with the change, my previous objection was that the change was breaking, but so many of these are breaking changes no, it really doesn't matter. Anyone wanting to use Version 7 will need to redo their mapping.

**Comment by gerardmoloneyetpartners** [comment]
![EquipmentLevelType](https://user-images.githubusercontent.com/54049079/89280800-bab58080-d67b-11ea-9fec-f1601141d442.PNG)

I am not entirely an expert on the coding side of things, however when I attempt to write an equipmentclass record as above, line 53 appears to me to align with ISA95 Part2 Table 40 & 44 for EquipmentClass and Equipment attributes are specified (where EquipmentLevel is listed), however the resolver currently says that this is invalid, and as you can see, it is resolves when written differently (line 63-66). 

Hopefully I am on the right track here with a solid example. @Dennis-Brandl and @CharlieG021163 
I guess this is already accepted at any rate, and the specifics of how will be done when the pull request is fully committed.
I'm happy, seeing as I am finding this issue today, and seeing we already covered it.
